src/net/java/sip/communicator/impl/protocol/jabber/extensions/colibri/ColibriConferenceIQ.java

```diff
@@ -2271,13 +2361,64 @@ public void setDisplayName(String displayName)
         }
 
         /**
+         * Sets the transport element of this <tt>ChannelBundle</tt>.
+         * @param transport the transport to set.
+         */
+        public void setTransport(IceUdpTransportPacketExtension transport)
+        {
+            this.transport = transport;
+        }
+
+        /**
          * Sets the ID of this <tt>Endpoint</tt>.
          * @param id the ID to set.
          */
         public void setId(String id)
         {
             this.id = id;
         }
+
+        /**
+         * Sets the stats ID of this <tt>Endpoint</tt>.
+         * @param statsId the stats ID to set.
+         */
+        public void setStatsId(String statsId)
+        {
+            this.statsId = statsId;
+        }
+
+        /**
+         * Appends the XML <tt>String</tt> representation of this
```

Comment:
If you reference other classes, please start using {@link ClassName} instead of <tt>, this creates navigatable links and is safe for refactoring as IDEs will recognize the reference.